### ISSUE TITLE
Peter Rasborg

### ISSUE THREAD

**Issue by Peter-Rasborg** [integrations/microsoft-teams#308]
The VIP Software 

Let me know what's next in the kitchen...

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Peter-Rasborg** [comment]
I've never been so disappointing in my life I wake up and I find a turn off my stuff the emails I am trying to get it sorted. My phone gets hacked shut down my laptop gets hacked and I called the only team member phone number that I have and they leave a message and ask how / why anyone would do this to ![image](https://user-images.githubusercontent.com/72441904/226573784-0ad584b3-66cd-4ff0-87d6-51f479035177.jpeg)![image](https://user-images.githubusercontent.com/72441904/226573824-4e363346-e9d7-41ac-be6a-6d1feb9869d5.jpeg)![image](https://user-images.githubusercontent.com/72441904/226573853-3ac251f4-82b2-488c-b97e-4994977ed9cd.jpeg)
And there were hundreds more of every password to my phone or laptop is ever logged into that was getting reset .

So I have absolutely no idea what was going on and I didn't know that I did anything wrong. I deserve to get hacked or messed with if there was an issue hours, appreciate it it was addressed to my face if I started locking my document sending unknown amount of emails to I unknown places so I haven't had a chance to find out so I basically thought or think that I just got mugged. Don't get me wrong. I appreciate anybody that wants to teach you anything or anything like that, but not communicating with anybody is definitely not a way that I've ever exercised. Any professionalism. 
My iPhone password and Apple ID everything you name it Eva's been changed and it was a big deal and I don't know why so that's where I'm at work more than welcome 

Best Regards 

Peter Rasborg
